spec/raise_message_cop_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require_relative '../lib/cops/raise_message_cop.rb'
+RSpec.describe RuboCop::Cop::CustomCops::RaiseMessageCop do
+    context 'corrector' do
+        subject(:cop) { described_class.new }
+        it 'it should register an offense if there is an empty raise present' do
+            expect_offense(<<~RUBY)
+              module D
+                class A
+                    def B
+                      raise
+                      ^^^^^ Raise should be accompanied by a message/argument.
+                    end
+                end
+              end
+            RUBY
+        end
+        it 'it should not register an offense if raise is accompanied by an argument/message' do
+          expect_no_offenses (<<~RUBY)
+          module D
+            class A
+                def B
+                  raise "A message with context to the raised error."
+                end
+            end
+          end
+
+          RUBY
+
+          expect_no_offenses (<<~RUBY)
+          module D
+            class A
+                def B
+                  raise StandardError, 'message'
```

Comment:
Can you add a test case with inline condition aswell
1. raise StandardError, 'message' unless status # Should not raise offense
2. raise unless status # should show offense